service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -59,4 +59,8 @@ public HearingBooking getHearingBookingByUUID(List<Element<HearingBooking>> hear
             .findFirst()
             .orElse(null);
     }
+
+    public void filterPreviousHearings(List<Element<HearingBooking>> before, List<Element<HearingBooking>> after) {
+        after.removeIf(before::contains);
+    }
```

Comment:
So a thought - it looks for me when I edit the hearing in past, it's not going to be removed from list, thus validation will happen for it, i.e. I could modify Magistrate name as there was typo in the name. 

Validation would fail for that entry. 

It essentially makes past hearing not editable does it? Maybe that's what the business would expect? Should we ask them?